### DISCUSSION TITLE
Copilot not detected

### DISCUSSION THREAD

**Discussion by TarunavBA** [community/community#6980] [Copilot Conversations]
Hey @github team, I have install `GitHub Copilot` in my windows 7 laptop, vscode latest version, and I see that neither the `GitHub copilot extension wants to signup using GitHub` comes nor I can see the `Copilot logo` in the footer of VSCode.

**Comment by davecheney** [comment]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by TarunavBA** [reply]
@davecheney I'm on my father's windows 7 enterprise Verison.

**Reply by TarunavBA** [reply]
> Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

@davecheney Is there any workaround for this?

**Reply by davecheney** [reply]
@TarunavBA i see no connections with your token to the copilot servers. Something on your network or computer is blocking the connection.

**Reply by TarunavBA** [reply]
@davecheney now what can I do? Please suggest a workaround. I want to use GitHub copilot. 🙁
Is there no way I can use it?

**Reply by TarunavBA** [reply]
@davecheney I didn't even get the prompt for singing in with github

**Reply by davecheney** [reply]
I think this is a networking issue on your end. Sorry, I am not able to assist further.

**Reply by TarunavBA** [reply]
So @davecheney will I be able to use copilot once my network gets good?

**Reply by TarunavBA** [reply]
Please @davecheney tell the answer for this and I won't ask further questions. 

> So @davecheney will I be able to use copilot once my network gets good?
